packages/core/getUserAvailability.ts

```diff
@@ -215,11 +258,13 @@ const _getUserAvailability = async function getUsersWorkingHoursLifeTheUniverseA
   let eventType: EventType | null = initialData?.eventType || null;
   if (!eventType && eventTypeId) eventType = await getEventType(eventTypeId);
 
+  const isTeamEvent = eventType?.schedulingType !== null;
```

Comment:
In comment about "_getCurrentSeats now takes in 5 parameters" I suggest "const isTeamEvent" logic should be moved inside "_getCurrentSeats", but then I am also thinking:

I think it's better to be explicit about when something is a team event:
```
const isTeamEvent = eventType?.schedulingType === SchedulingType.COLLECTIVE  || eventType?.schedulingType === SchedulingType.ROUND_ROBIN || eventType?.schedulingType === SchedulingType.COLLECTIVE
```

My logic for this is that currently `SchedulingType` indeed shows only scheduling types of teams, but what if someone adds a `SchedulingType` that is non-team? Then this logic would potentially break.

Reply:
We can also directly filter on hosts as only team events have hosts but that might change in the future so using hosts and schedulingType